### PR TITLE
NISP-1928: Add Terms and Conditions messaging to messages file

### PR DESCRIPTION
**Pull request by fred-jackson**

### CODE REVIEW

**Comment by dspork** [comment]
I'd have all the code and messages added in at once, so I won't merge the request and just leave the branch open until that's done too?

**Comment by fred-jackson** [comment]
@dspork - it was just so that the code could be checked before I start on the next part of the task, creating the terms and conditions page. I wouldn't merge it until everything has been created and tested :smiley:

**Comment by dspork** [comment]
Well you haven't written any "code" yet so ;)

**Comment by fred-jackson** [comment]
Now I have, so ner! :stuck_out_tongue_closed_eyes:

File: app/uk/gov/hmrc/nisp/views/termsAndConditions.scala.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import uk.gov.hmrc.nisp.controllers.routes
+@import uk.gov.hmrc.nisp.controllers.auth.NispUser
+@import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
+
+@()(implicit request: Request[_], user: NispUser, partialRetriever: CachedStaticHtmlPartialRetriever)
```
Comment (dspork) [inline app/uk/gov/hmrc/nisp/views/termsAndConditions.scala.html]: This page shouldn't have a `NispUser` as they won't necessarily be logged in.

**Comment by dspork** [comment]
Where's the controller method and route yo?

**Comment by fred-jackson** [comment]
@dspork - I have finished :smile: 

@InduTest and @swaistle - looks like this will be ready to test, once @dspork has reviewed my 'code'.

**Comment by dspork** [comment]
Your 'code' is fine @fred-jackson. Once tested then you can :shipit: @InduTest / @swaistle

**Comment by fred-jackson** [comment]
Merged and branch deleted

**Pull request merged**